src/main/java/com/taf/business/pages/DashboardsPage.java

```diff
@@ -0,0 +1,41 @@
+package com.taf.business.pages;
+
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.CollectionCondition.itemWithText;
+import static com.codeborne.selenide.CollectionCondition.sizeGreaterThan;
+import static com.codeborne.selenide.Selenide.$$x;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class DashboardsPage {
+    private static final String PAGE_HEADER = "//span[@title='All Dashboards']";
```

Comment:
For constants, it is recommended to use static because it will save memory in case we will create more than one object using a class with no static constants.